### PR TITLE
add task 11

### PR DESCRIPTION
**Pull request by feroxes**

### CODE REVIEW

**Comment by AMashoshyna** [comment]
could you check please? I cant't see the files in the PR

**Review by AMashoshyna — DISMISSED** [review]
no files added

**Comment by feroxes** [comment]
done

**Comment by AMashoshyna** [comment]
Thanks! Do you have a link to demo? Could you please add it as comment to PR?

**Comment by feroxes** [comment]
https://feroxes.github.io/task_11/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+			</div>
+	</section>
+	<section class="one">
+		<div class="upload">
+			<p class="cirkle">1</p>
+			<h3>Upload your documents</h3>
+			<p class="types">(Doc, Docx, Text, PDF, Markdown)</p>
+
+		</div>
+			<button class="main_btn upload_btn">Upload a new file</button>
```
Comment (AMashoshyna) [inline index.html]: there is a dedicated element for file upload <input type="file">
when clicked, it causes browser to open file picker. Check out here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+	<div class="col-6">
+		<div class="details">
+			<div class="blank"><!--<p><img src="https://png.icons8.com/material/26/6c748d/file.png"></p>--> <p>Your order</p></div>
+			<div>
+				<ul class="list">
+					<li><span>Items</span></br></br>2 documents</li>
+					<li><span>Details</span></br></br>2 languages</li>
+					<li><span>Price</span></br></br>$ 696.06</li>
+					<li><span>Delivery</span></br></br>48 hours*</li>
+				</ul>
```
Comment (AMashoshyna) [inline index.html]: this looks like a perfect case to use table element (list usually contains similar elements)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+	<section class="two">
+		<div class="upload">
+			<p class="cirkle2">2</p>
+			<h3>Choose youre languages pairings</h3>
+		</div>
+		<div class="language">
+			<div class="from">
+			<p>From</p>
+			<div class="form_select">
+				<select type="text" name="capacity" required="">
```
Comment (AMashoshyna) [inline index.html]: `required` is boolean - it is true if it is present. so, just `required` is enough 

also, `select` element does not support `type` attribute. what is the purpose of it here?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+			</div>
+			<div class="f_name code">
+			<p>code</p>
+			<input type="password" name="name" maxlength="3" class="" required="">
+			</div>
+		</div>
+		<div class="last_step">
+			<input id="checkBox" type="checkbox">
+			<p>I agree to the <a href="#">Terms &#38; Condiions</a></p>
+			<button class="act_btn">Place order</button>
```
Comment (AMashoshyna) [inline index.html]: `button` element without explicit type defaults to `submit` type. which means that every button will attempt to send a form to the server.

**Comment by AMashoshyna** [comment]
let's start by fixing html:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fferoxes.github.io%2Ftask_11%2F
let me know, if anything from the comments is unclear

**Comment by feroxes** [comment]
everything understandably, but don`t understand why I have this error in validator: 
Element div not allowed as child of element label in this context. (Suppressing further errors from this subtree.)

**Comment by feroxes** [comment]
should I fix that?

**Comment by AMashoshyna** [comment]
@feroxes since `label` element is designed to hold some text (caption for input, for example), it should not contain block elements like `div`. Check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) under "Permitted content". If you need a `div` as a wrapper for styling purposes only, you could use `span` or style the element directly (without wrapper)

**Comment by cidre** [comment]
<label>  had child <div> on @lempiy slides on chapter "Label trick" :)

**Comment by AMashoshyna** [comment]
@feroxes could you update the demo please?
there is still this thing with <input type="file"> instead of a button.
also it is better to avoid committing service files like .DS_Store. You can remove the file and add it to .gitignore file, so it does not happen again.

File: index.html [hunk 5]
```diff
@@ -12,25 +12,39 @@
...
+						<td>48 hours*</td>
+					</tr>
+				</table>
+				<!-- <ul class="list">
 					<li><span>Items</span></br></br>2 documents</li>
 					<li><span>Details</span></br></br>2 languages</li>
 					<li><span>Price</span></br></br>$ 696.06</li>
 					<li><span>Delivery</span></br></br>48 hours*</li>
-				</ul>
+				</ul> -->
```
Comment (AMashoshyna) [inline index.html]: How about using [`address`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address) element?
Or at least use anchor element (`<a>`) with href="mailto:help@qtest.com". check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) under *Creating an email link* and *Creating phone link*

File: index.html [hunk 6]
```diff
@@ -264,7 +278,7 @@ <h3>Sing up and choose a payment method</h3>
 			<p>Our main office</p>
 		</div>
 		<div class="adress">
-			<p>Dubai, UAE </br>P.O. Box: 502735</br>Phone: +971 (0) 44278128</p>
+			<p>Dubai, UAE <br>P.O. Box: 502735<br>Phone: +971 (0) 44278128</p>
```
Comment (AMashoshyna) [inline index.html]: Same here: use semantic `address` element or at least use `href` attribute with email/phone

**Comment by lempiy** [comment]
I've made mistake. As you understood, I'm not a real сварщик. :stuck_out_tongue:

File: index.html [hunk 7]
```diff
@@ -45,28 +59,28 @@
...
 	</div>
-</section>
+</div>
 <form class="main">
 	<section class="select">
 		<h3>Select what you want to translate</h3>
 			<div class="form">
 				<label for="website">
-					<input hidden type="radio" name="radio1" id="website" value="website">
+					<input hidden type="radio" required name="radio1" id="website" value="website">
```
Comment (AMashoshyna) [inline index.html]: there is a problem with `required` attribute on `hidden` elements: it does not work :) the reason is you cannot require user to interact with input that the user may тot see. check it: you can still send the form without checking this field.
it is better to choose some other way to visually hide an element, like moving it off-screen or other. anyway, hide it with css, not html.

File: index.html [hunk 8]
```diff
@@ -77,7 +91,7 @@ <h3>Upload your documents</h3>
 			<p class="types">(Doc, Docx, Text, PDF, Markdown)</p>
 
 		</div>
-			<button class="main_btn upload_btn">Upload a new file</button>
+			<button class="main_btn upload_btn" type="button">Upload a new file</button>
```
Comment (AMashoshyna) [inline index.html]: Use `<input type="file">`

**Comment by cidre** [comment]
@lempiy  It's ok. Thanks for slides and explanations. :)

**Comment by feroxes** [comment]
https://feroxes.github.io/task_11/ is updated. Continue to work

**Comment by feroxes** [comment]
Fix some problems. Continue to work.

**Comment by feroxes** [comment]
fix upload button.

**Comment by feroxes** [comment]
All done))
Check please

**Review by lempiy — APPROVED** [review]

**Review by xandzia — APPROVED** [review]

**Pull request merged**